3ds Max/Max2Babylon/Exporter/BabylonExporter.Skeleton.cs

```diff
@@ -61,67 +61,42 @@ private List<IIGameNode> GetRevelantNodes(IIGameSkin skin)
                 return revelantNodesBySkin[skin];
             }
 
-            List<IIGameNode> revelantNodes = new List<IIGameNode>();
             List<IIGameNode> bones = GetBones(skin);
 
-            if(bones.Contains(null))
+            if (bones.Count == 0)
             {
-                RaiseError("Skin has bones that are outside of the exported hierarchy.", logRank);
-                RaiseError("The skin cannot be exported", logRank);
+                RaiseWarning("Skin has no bones.", logRank);
                 return new List<IIGameNode>();
             }
-            
-            // for each bone of the skin, add their parents in the revelantNodes list.
-            foreach (IIGameNode bone in bones)
+
+            if (bones.Contains(null))
             {
-                if (!revelantNodes.Contains(bone))
-                {
-                    revelantNodes.Add(bone);
-                    IIGameNode currentNode = bone.NodeParent;
-                    while (currentNode != null)
-                    {
-                        if (!revelantNodes.Contains(currentNode))
-                        {
-                            revelantNodes.Add(currentNode);
-                            currentNode = currentNode.NodeParent;
-                        }
-                        else // the node and its parents are already in the list
-                        {
-                            currentNode = null;
-                        }
-                    }
-                }
+                RaiseError("Skin has bones that are outside of the exported hierarchy.", logRank);
+                RaiseError("The skin cannot be exported", logRank);
+                return new List<IIGameNode>();
             }
 
-            // return an empty list if there is more than one root
-            List<IIGameNode> rootNodes = revelantNodes.FindAll(node => node.NodeParent == null);    // should contains only one node
-            if (rootNodes.Count > 1)
+            //return a list of all bones that are root
+            IIGameNode rootNodes = GetCommonAncestor(bones);
+            if (rootNodes == null)
             {
                 string rootNames = "";
-                foreach(IIGameNode root in rootNodes)
-                {
-                    rootNames += $" {root.Name}";
-                }
                 RaiseError($"More than one root node for the skin: {rootNames}", logRank);
```

Comment:
Can we update the debug outputs and function docstring to reflect the change in logic?